### ISSUE TITLE
flash-platform

### ISSUE THREAD

**Issue by matrix-1996** [adobe/flash-platform#1]
Hello, I want to tell you that the repository  is empty, please explain why it was created

**Comment by wmaddox** [comment]
I suspect it was created to hold some subprojects that were instead placed at ‘top-level’, for example, the Actionscript 3 virtual machine under adobe/avmplus.  It looks like most of Adobe’s open-source presence on GitHub has moved here:

http://adobe.github.io/

 

Best regards,

 

--Bill

**Comment by filmaj** [comment]
Hey @ryanstewart, question: can we delete this repo? It is empty and I don't think it provides much value as-is.

Thanks,
Fil

**Comment by ryanstewart** [comment]
@filmaj yeah, I have no idea what this repo was intended to be. So definitely okay to delete, I think. Thanks for checking.
